server/routes/helpCentreFrontend.ts

```diff
@@ -12,7 +20,55 @@ import {
 
 const router = Router();
 
-router.use(withIdentity(), async (_: Request, res: Response) => {
+export const getCookiesOrEmptyString = (req: express.Request) =>
+	req.header('cookie') || '';
+
+router.use(withIdentity());
+
+router.use('/diagnostic-information', async (req, res, next) => {
```

Comment:
Most of the fetch requests in MMA are done on the page component via the server (see `api.ts`) - but I don't have a problem with doing it this way (except maybe it's slightly hidden away in this file)